casadi/interfaces/sundials/idas_interface.cpp

```diff
@@ -374,6 +398,9 @@ namespace casadi {
       THROWING(IDAAdjInit, m->mem, steps_per_checkpoint_, interpType);
     }
 
+    // Initialize root function
+    THROWING(IDARootInit, m->mem, 1, root);
```

Comment:
This only handles one expression right now, so that would need to be fixed if we want to keep this.